UI/window-missing-files.cpp

```diff
@@ -49,7 +49,7 @@ MissingFilesPathItemDelegate::MissingFilesPathItemDelegate(
 
 QWidget *MissingFilesPathItemDelegate::createEditor(
 	QWidget *parent, const QStyleOptionViewItem & /* option */,
-	const QModelIndex &index) const
+	const QModelIndex &) const
```

Comment:
`QStyleOptionViewItem` and `QModelIndex` can be removed

Reply:
This only works for slots, and this is not a slot.